question/views.py

```diff
@@ -1,13 +1,20 @@
 from django.shortcuts import get_object_or_404
 from django.utils import timezone
-from rest_framework import status
+from django.http import Http404
+from rest_framework import generics, permissions, status
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
+<<<<<<< HEAD
+from question.models import Question, Comment
+from question.serializers import QuestionSerializer, CommentSerializer
+from question.permissions import IsOwnerOrReadOnly
+=======
 from meetup.models import Meeting
 from question.models import Question, Vote
 from question.serializers import QuestionSerializer, VoteSerializer
+>>>>>>> 54e3da93010e1083a4b17abf2b76bcfc556c02a8
```

Comment:
Please remove the merge conflict markers

Reply:
The conflict markers are also part of the reason why travis is failing

Reply:
Sure. Just removed them.

Reply:
Hello @mwinel, the branch you plan to merge with is current stable (develop), I would recommend doing an upstream pull from it then add new code to it, else with the current Travis report, a merge would render develop unstable.